tests/manual/manual-create-from-github.R

```diff
@@ -1,47 +1,68 @@
-load_all()
+devtools::load_all()
```

Comment:
Maybe it would useful to have a README.md in the tests/manual directory explaining how you typically run these.

e.g. do you use `source()`, do you just run them line by line in the console, is there setup you need to do prior to running etc.

It doesn't need to be long, a few sentences would probably be ok.